config/settings/common.py

```
@@ -52,6 +52,12 @@
     _ADMIN_DJANGO_APP = ['django.contrib.admin']
     _ADMIN_OAUTH2_APP = []
 
+ES_APM_ENABLED = env.bool('ES_APM_ENABLED')
+
+_ES_APM_APP = []
```

Should we set the default to `False`, just in case?

---

I will make sure the environments are updated before merging. I think it shouldn't silently default to being switched off, so that we are sure it is being consciously configured.